sql/moz-fx-data-marketing-prod/ga_derived/www_site_metrics_summary_v2/query.sql

```diff
@@ -23,7 +23,7 @@ WITH site_data AS (
   FROM
     `moz-fx-data-marketing-prod.analytics_313696158.events_*`
   WHERE
-    event_date = FORMAT_DATE('%Y%m%d', @submission_date)
+    _table_suffix = FORMAT_DATE('%Y%m%d', @submission_date)
```

Comment:
Although `_TABLE_SUFFIX` isn't case sensitive and `_table_suffix` works, I'd suggest still capitalizing it like it is in the BigQuery docs, both for consistency (it's capitalized in most instances in existing ETLs) and to help it stand out as not being a normal column.
```suggestion
    _TABLE_SUFFIX = FORMAT_DATE('%Y%m%d', @submission_date)
```